performance_manager/startup.py

```diff
@@ -138,11 +138,21 @@ def iteration() -> None:
 
         try:
             process_static_tables(db_manager)
-            process_vehicle_positions(db_manager)
-            process_trip_updates(db_manager)
-            process_full_trip_events(db_manager)
-            process_dwell_travel_times(db_manager)
-            process_headways(db_manager)
+
+            min_ts_processed, max_ts_processed = process_vehicle_positions(
+                db_manager
+            )
+            min_ts, max_ts = process_trip_updates(db_manager)
+
+            min_ts_processed = min(min_ts_processed, min_ts)
+            max_ts_processed = max(max_ts_processed, max_ts)
+
+            if max_ts_processed > 0:
+                process_full_trip_events(
+                    db_manager, min_ts_processed, max_ts_processed
+                )
+                process_dwell_travel_times(db_manager)
+                process_headways(db_manager)
```

Comment:
capture min/max timestamp values from level 0 table processing. if `max_ts_processed` is 0 this means that no new data was added by level 0 processing